packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -216,6 +217,33 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
     );
   }
 
+  Future<int?> _getMyCount(
+    final String userId,
+    final UserProductSearchType type,
+    final OpenFoodFactsLanguage language,
+  ) async {
+    final UserProductSearchQueryConfiguration configuration =
+        UserProductSearchQueryConfiguration(
+      type: type,
+      userId: userId,
+      pageSize: 1,
+      language: language,
+      fields: [],
```

Comment:
You must specify the type: `<ProductField>[]`.
You should not have any warning before you push.